synfig-core/src/modules/mod_svg/svg_parser.cpp

```diff
@@ -505,16 +512,134 @@ Svg_parser::build_outline(xmlpp::Node* root, Style style, const std::list<BLine>
-		else build_param (child_outline->add_child("param"),"sharp_cusps","bool","false");
-		if(stroke_linecap.compare("butt")==0){
-			build_param (child_outline->add_child("param"),"round_tip[0]","bool","false");
-			build_param (child_outline->add_child("param"),"round_tip[1]","bool","false");
-		}else{
-			build_param (child_outline->add_child("param"),"round_tip[0]","bool","true");
-			build_param (child_outline->add_child("param"),"round_tip[1]","bool","true");
+		return child_outline;
+	};
+	if (!is_advanced_outline) {
+		for (const BLine& bline : k) {
+			xmlpp::Element *child_outline = create_layer_node_with_common_info(bline);
+			child_outline->set_attribute("type","outline");
+			if(stroke_linejoin.compare("miter")==0) build_param (child_outline->add_child("param"),"sharp_cusps","bool","true");
+			else build_param (child_outline->add_child("param"),"sharp_cusps","bool","false");
+			if(stroke_linecap.compare("butt")==0){
+				build_param (child_outline->add_child("param"),"round_tip[0]","bool","false");
+				build_param (child_outline->add_child("param"),"round_tip[1]","bool","false");
+			}else{
+				build_param (child_outline->add_child("param"),"round_tip[0]","bool","true");
+				build_param (child_outline->add_child("param"),"round_tip[1]","bool","true");
+			}
+			build_param (child_outline->add_child("param"),"homogeneous_width","bool","true");
+		}
+	} else {
+		const std::map<std::string, int> linejoin_map = {
+			{"miter", 0}, // Advanced_Outline::CuspType::TYPE_SHARP
+			{"round", 1}, // Advanced_Outline::CuspType::TYPE_ROUNDED
+			{"bevel", 2}, // Advanced_Outline::CuspType::TYPE_BEVEL
+			{"arcs", 0},       // not equivalent
+			{"miter-clip", 0}, // not equivalent
+		};
+		const std::map<std::string, int> linecap_map = {
+			{"round", 1}, // WidthPoint::TYPE_ROUNDED
+			{"square", 2},// WidthPoint::TYPE_SQUARED
+			{"butt", 4}   // WidthPoint::TYPE_FLAT
+		};
+		int linejoin_value = linejoin_map.at("miter");
+		{
+			auto iter = linejoin_map.find(stroke_linejoin);
+			if (iter != linejoin_map.end())
+				linejoin_value = iter->second;
+		}
+		int linecap_value = linecap_map.at("butt");
+		{
+			auto iter = linecap_map.find(stroke_linecap);
+			if (iter != linecap_map.end())
+				linecap_value = iter->second;
+		}
+
+		float path_length = 0.f;
+		if (!k.empty())
+		{
+			std::vector<synfig::BLinePoint> v;
+			for (const auto& point : k.front().points) {
+				v.push_back(BLinePoint());
+				v.back().set_vertex(synfig::Point(point.x, point.y));
+				v.back().set_tangent1(synfig::Point(point.radius1, point.angle1));
+				v.back().set_tangent2(synfig::Point(point.radius2, point.angle2));
+				v.back().set_split_tangent_angle(point.split_angle);
+				v.back().set_split_tangent_radius(point.split_radius);
+			}
+
+			path_length = scale_factor * bline_length(v, k.front().loop, nullptr);
+		}
+
+		std::vector<float> dashes;
+
+		float dash_sum = 0.f;
+		for (String token : tokenize(stroke_dasharray, " ,\0x09\0x0a\0x0d"))
```

Comment:
Is that correct? Looks like you want to escape `<tab>, <cr>, <lf>`.
In that case it should be `\x09` without zero.
https://en.cppreference.com/w/cpp/language/escape

Reply:
Fixed!